lib/gpio.js

```diff
@@ -1,26 +1,46 @@
-var wpi = require('wiring-pi');
+var DEVICE_NAME = 'gpio';
+var gpioLibrary = require('./gpio-la-mock');
 var gpios = null;
 
-function updatePinStates() {
+function fetchPinByIndex(callback, pinsToRead, pinIdx) {
```

Comment:
I believe it's more standard in `nodejs` libraries to make the callback the final parameter. Would you be willing to reorder these parameters to match that convention?